requirements.txt

```diff
@@ -1,34 +1,68 @@
-cycler==0.10.0
+#
+# This file is autogenerated by pip-compile with python 3.6
+# To update, run:
+#
+#    pip-compile --output-file=-
+#
+--find-links https://download.pytorch.org/whl/torch_stable.html
+
+cycler==0.11.0
```

Comment:
cycler 0.10.0 changed to 0.11.0

Reply:
Strangely, `pip-compile` **sometimes** introduced the different package versions when run multiple times in the same environment on the same `requirements.in` file.

I've pushed a correctly-generated `requirements.txt` file in https://github.com/Deep-MI/FastSurfer/pull/81/commits/d6298868ca2e03b16d1ba68a2f00b8f538cc169b